### DISCUSSION TITLE
Is it possible to bump the @tanstack/react-router-server vinxi dependency version to resolve CVE-2024-23331 / CVE-2024-31207 /  CVE-2023-49293?

### DISCUSSION THREAD

**Discussion by mrfigg** [TanStack/router#1429] [General]
As the title says, is it possible to bump the [@tanstack/react-router-server](https://github.com/TanStack/router/tree/main/packages/react-router-server) [vinxi](https://github.com/nksaraf/vinxi) dependency version to resolve a number of CVEs with [vite](https://github.com/vitejs/vite)?

I understand this could be a big ask, but I figure it couldn't hurt to ask.

https://github.com/advisories/GHSA-c24v-8rfc-w8vw
https://github.com/advisories/GHSA-8jhw-289h-jh2g
https://github.com/advisories/GHSA-92r3-m2mg-pj97

**Comment by schiller-manuel** [comment]
bump to which versions? feel free to submit a PR
